java/arcs/sdk/android/storage/ServiceStore.kt

```diff
@@ -176,6 +189,7 @@ class ServiceStore<Data : CrdtData, Op : CrdtOperation, ConsumerData>(
     fun onLifecycleDestroyed() {
         serviceConnection?.disconnect()
         storageService = null
-        scope.coroutineContext[Job.Key]?.cancelChildren()
+        channel = null
+        scope.cancel()
```

Comment:
should this re-init the channel too? to ensure it's canceled correctly before the scope is canceled?